org.tech.JTDX.yml

```diff
@@ -0,0 +1,66 @@
+app-id: org.tech.JTDX
```

Comment:
`tech.org` seems to be unrelated to this package.

this would make the app-id invalid.

Reply:
in readme https://github.com/jtdx-project/jtdx#readme

Project Web site:

http://www.jtdx.tech/

Project mailing  list:

https://jtdx.groups.io/g/main

app-id : tech.jtdx.JTDX ?